capstone/scripts/extract_cites.py

```diff
@@ -195,38 +205,46 @@ def resolve_full_cite(eyecite_cite):
             target_cases = []
             target_case_id = None
 
-        # collect pin cites
-        pin_cites = []
-        for cite in cluster:
-            extra = {}
-            if getattr(cite.metadata, 'parenthetical'):
-                extra['parenthetical'] = cite.metadata.parenthetical
-            if getattr(cite.metadata, 'pin_cite'):
-                page = cite.metadata.pin_cite or ''
-                if page.startswith('at '):
-                    page = page[3:]
-                extra['page'] = page
-            if extra:
-                pin_cites.append(extra)
-
-        # create new cite
-        normalized_forms = first_cite.normalized_forms
-        # NOTE if adding any fields here, also add to cite_key()
-        extracted_cite = ExtractedCitation(
-            **normalized_forms,
-            reporter=first_cite.corrected_reporter(),
-            category=category,
-            cited_by=case,
-            target_case_id=target_case_id,
-            target_cases=target_cases,
-            groups=first_cite.groups or {},
-            metadata=metadata,
-            pin_cites=pin_cites,
-            weight=weight,
-            year=first_cite.year,
-        )
-
-        found_cites.append(extracted_cite)
+        for opinion_id, opinion_cluster in opinion_clusters.items():
+            opinion_includes_cite = {}
+            for cite in opinion_cluster:
```

Comment:
Loop isn't necessary anymore, right? You just end up processing `opinion_cluster[0]` for each cluster.

Reply:
yeah -- we only process opinion_cluster[0] to pull the weight, but use cluster[0] to pull the other metadata. we loop through the opinion_cluster for pin cites.